Calculate domestic shipping

elusiveshiba opened the pull request:
Calculate domestic shipping if fixed rates are not provided in config file.
Origin postcode added to config file - needed for calculating domestic shipping rates.
Fixed bug when calculating price_combined_total (strings were added resulting in 123+456=123456)

elusiveshiba committed:
Calculate domestic shipping if fixed rates are not provided in config file.
Origin postcode added to config file - needed for calculating domestic shipping rates.
Fixed bug when calculating price_combined_total (strings were added resulting in 123+456=123456)

SomeoneWeird commented on src/routes/shipping.ts:
```diff
@@ -86,7 +97,7 @@ async function handleShippingCalc(req: Request, res: Response): Promise<void> {
       price_shipping_and_handling_only: s.price.toString(),
       price_product_only: config.editions[sku].price.toFixed(),
-      price_combined_total: (config.editions[sku].price + s.price).toString(),
+      price_combined_total: (Number(config.editions[sku].price) + Number(s.price)).toString(),
```
I'd swap these `Number()` calls out to be explicitly `parseFloat` so they don't accidentally get rounded/truncated - we should probably also do the parsing immediately after they get loaded (or we validate the config file and throw if they're not numbers?) as we're calling `.toFixed()` on  `config.editions[sku].price` which will throw an error if it's a string, too.

elusiveshiba replied:
Thanks, I've updated this now so s.price uses `parseFloat`. `config.editions[sku].price` is actually number to begin with so didn't need to be calling `Number()` on it

elusiveshiba committed:
Updated readme with new config, specifies prices should all be in DOGE.
Require postcode to be sent and environment variable.
toDogePlusHandling now rounds to nearest 0.69 for minimal impact on final price.
Calculated domestic shipping rate now converted to DOGE in line with international shipping

elusiveshiba committed:
Using parseFloat instead of Number. No need to convert config.editions[sku].price as it's already of type number. Removed unused import as fixed rate shipping prices in config are now provided in DOGE

elusiveshiba commented:
Ready for review @SomeoneWeird @patowens @blendtwenty

elusiveshiba committed:
Added dev config file, API key moved to env var
Changed from NODE_ENV to CONFIG_ENV for config definition
Ignore .env

elusiveshiba committed:
Port moved to env var, defaults to 3333 if not found

SomeoneWeird committed:
Add vercel.json
